plugins/DiskSize/plugin.rb

```diff
@@ -18,12 +20,13 @@ def initialize
   end
 
   def store(fqdn)
+
     # Pull node information from cache as a ruby hash
     node_info = @cache.get(fqdn)
 
     # Check for valid data
     if node_info['disk_sizes'].nil? || node_info['disk_sizes'] == 'unknown'
-      raise "No disk_sizes information for #{fqdn}"
+      MyLog.log.warn "DiskSize: No disk_sizes information for #{fqdn}"
```

Comment:
@alxngyn Yeah, I think that when I replaced the _raise_ it somehow triggered the rescue. I returned the _raise_ and removed the log and now the tests seem to be passing. Can you check if that works for you? I wonder why those errors didn't show up originally...